docs/dpsg/getting-started/config.md

```diff
@@ -0,0 +1,65 @@
+---
+id: config
+title: Configuration
+sidebar_label: Configuration
+description: How to configure DPSG
+---
+
+## Connecting your Falcon App
+
+All the setup is done our side. All you need to do is configure your Falcon App to use DPSG and your good to go.
+
+To connect your app to DPSG you'll just need to make a few simple configuration changes.
+
+:::note Get your DPSG credentials?
```

Comment:
@hobadams nobody should be contacting us. It is possible to get the DPSG credentials via `dcloud payments:env:info` (using the same DC auth0 token).

Reply:
on it